src/java/org/jivesoftware/openfire/admin/JDBCAdminProvider.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * $Revision$
+ * $Date$
+ *
+ * Copyright (C) 2005-2008 Jive Software. All rights reserved.
```

Comment:
Done, hehe older than that, my local git repo has:

commit 5a7e21b3f995077746a499dc7d2ecb5b82b8954b
Author: Robert Marcano ....
Date:   Wed Nov 5 21:35:04 2008 +0000